packages/leancode_hooks/README.md

```diff
@@ -0,0 +1,16 @@
+# leancode_hooks
+
+[Hooks][flutter-hooks] we often use, all gathered in one place for better
+discoverability and consistent versioning.
+
+[![leancode_hooks pub.dev badge][pub-badge]][pub-badge-link]
+
+[pub-badge]: https://img.shields.io/pub/v/leancode_hooks
+[pub-badge-link]: https://pub.dev/packages/leancode_hooks
+
+## Usage
+
+This package exports [`package:flutter_hooks`][flutter_hooks] so you won't have
+to depend on it.
```

Comment:
Could we have a list of the hooks we can use? Table with function name and a description would be very handy.

Reply:
I think it's a good idea but it'd require more mainteneance work.

For `v0.0.1` I'd prefer a simple markdown list with links to files.